flume/conf/cygnus.conf.template

```diff
@@ -80,6 +80,16 @@ cygnusagent.sinks.hdfs-sink.attr_persistence = column
 cygnusagent.sinks.hdfs-sink.hive_host = x.y.z.w
 # Hive port for Hive external table provisioning
 cygnusagent.sinks.hdfs-sink.hive_port = 10000
+# Kerberos-based authentication enabling
+cygnusagent.sinks.hdfs-sink.krb5_auth = false
+# Kerberos username
+cygnusagent.sinks.hdfs-sink.krb5_auth.krb5_user = krb5_username
+# Kerberos password
+cygnusagent.sinks.hdfs-sink.krb5_auth.krb5_password = xxxxxxxxxxxxx
+# Kerberos login file
+cygnusagent.sinks.hdfs-sink.krb5_auth.krb5_login_file = /usr/cygnus/conf/krb5_login.conf
```

Comment:
Yes, I refer to that. Reading your next comment, it seems that everying in flume/conf ends in /usr/cygnus/conf by the RPM magic, so it's ok.

NTC